cmd/podman/exec.go

```diff
@@ -19,6 +19,10 @@ var (
 			Name:  "privileged",
 			Usage: "Give the process extended Linux capabilities inside the container.  The default is false",
 		},
+		cli.BoolFlag{
+			Name:  "interactive, i",
+			Usage: "Keep STDIN open even if not attached",
```

Comment:
Lets add a note that this actually does not do anything like it is in the man page.

Reply:
fixed